### ISSUE TITLE
can I make it work with only local models (no API calls)?

### ISSUE THREAD

**Issue by HananInfiniGrow** [bhaskatripathi/pdfGPT#56]
for privacy and money constrains

**Comment by marc76900** [comment]
I'm also interested.

**Comment by bhaskatripathi** [comment]
Yes, it can be done and roughly it will take an effort of a week. But at present I am seriously short of bandwidth due to professional commitments. I will start looking at it in about a month's time.

**Comment by Anonym0us33** [comment]
Could you note the section where the model is called in python? I could link it in to a local model framework instead
